### PR TITLE
Pal 577 example configurator

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: example-model/src/main/java/uk/gov/gchq/palisade/example/configurator/ExampleConfigurator.java [hunk 1]
```diff
@@ -172,40 +176,24 @@ private void addSerialiser() {
...
-            resourceList.add(parentResource);
-        }
-        resourceList.get(resourceList.size() - 1).setParent(createSystemResource(str));
-
-        return resourceList.get(0);
-    }
-
-    private DirectoryResource addParentResource(final String path) {
-        return new DirectoryResource().id(path);
+    private void resolveParents(final Path path, final ChildResource childResource) {
```
Comment (ac74475) [inline example-model/src/main/java/uk/gov/gchq/palisade/example/configurator/ExampleConfigurator.java]: I would have expected to see some unit tests to prove this method works as expected, especially around edge cases.

Comment (dev930018) [reply example-model/src/main/java/uk/gov/gchq/palisade/example/configurator/ExampleConfigurator.java]: Since this will be compiled before running, any given test will be testing on the same environment it will run on.
The only real requirement is that there is SystemResource / at least one DirectoryResource / FileResource
